docs/tools/chrome-extension/linkedin-feature.md

```diff
@@ -0,0 +1,13 @@
+---
+id: linkedin 
+title: 'Linkedin Integration'
+sidebar_label: 'Linkedin Integration'
+keywords:
+  - Linkedin
+  - posting
+  - Linkedin project
+---
+
+You can add a repo to Linkedin as a project through the extension, the extension will get the name of the project and description filled autoamtically for you.  
+
```

Comment:
[suggestion]: This is a good start @a0m0rajab, but some things that are missing is:

- A statement that describes the benefit 
- A Call To Action. This is the statement that encourages people to how to get started. Check out the last sentence on the explainer page about [the Highlights feature](https://docs.opensauced.pizza/features/highlights/) for inspiration.

Reply:
Thanks, I just added that.